docs/core/additional-tools/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,118 @@
+
+## Prerequisites
+
+To complete this tutorial:
+
+* Install [.NET Core SDK 2.1.3 or later](https://www.microsoft.com/net/download)
+* Install your favorite code editor, if you haven't already.
+
+> [!TIP]
+> Need to install a code editor? Try [Visual Studio](https://visualstudio.com/downloads)!
+  
+## Use Microsoft XML Serializer Generator in a .NET Core console application 
+
+The following instructions show you how to use XML Serializer Generator in a .NET Core console application.
+
+### Create a .NET Core console application
+
+Open a command prompt and create a folder named *MyApp*. Navigate to the folder you created and type the following commands:
+
+```console
+dotnet new console
+```
+
+### Add a reference to the Microsoft.XmlSerializer.Generator package in the MyApp project.
+
+ Type:
+ 
+ ```console
+ dotnet add MyApp package Microsoft.XmlSerializer.Generator -v 1.0.0
+ ```
+ 
+### Verify changes to MyApp.csproj after adding the package
+
+Open your text editor and let's get started! We're still working from the *MyApp* directory we built the app in.
+
+Open *MyApp.csproj* in your text editor.
+
+After running the `dotnet add` command, the following lines are added to your *MyApp.csproj* project file:
+
+ ```xml
+ <ItemGroup>
+    <PackageReference Include="Microsoft.XmlSerializer.Generator" Version="1.0.0" />
+ </ItemGroup>
+ ```
+ 
+ ### Add another ItemGroup section for .NET Core CLI Tool support
+ 
+ Add the following lines after the `ItemGroup` section that we just inspected.
+ 
+ ```xml
+ <ItemGroup>
+    <DotNetCliToolReference Include="Microsoft.XmlSerializer.Generator" Version="1.0.0" />
+ </ItemGroup>
+ ```
+ 
+### Add a class in the application. 
+
+Open *Program.cs* in your text editor. Add the class named *MyClass* in *Program.cs*.
+
+```csharp
+public class MyClass
+{
+   public int Value;
+}
+```    
+
+### Create an `XmlSerializer` for MyClass.  
+
+Add the following line inside *Main* to create an `XmlSerializer` for MyClass:
+
+```csharp
+var serializer = new System.Xml.Serialization.XmlSerializer(typeof(MyClass));
+```    
+
+### Build and run the application
+
+Still within the *MyApp* folder, run the application via [`dotnet run`](../tools/dotnet-run.md)  and it will automatically load and use the pre-generated serializers at runtime.
+
+Type the following in your console window:
+
+ ```console
+ $ dotnet run
+ ```
+> [!NOTE]
+> [`dotnet run`](../tools/dotnet-run.md) calls [`dotnet build`](../tools/dotnet-build.md) to ensure that the build targets have been built, and then calls `dotnet <assembly.dll>` to run the target application.
```

Comment:
not sure if we should add a note similar to what we have in the end of the following tutorial given that dotnet run is only meant for dev scenarios:
https://docs.microsoft.com/en-us/dotnet/core/tutorials/using-with-xplat-cli

"Note that the commands and steps shown in this tutorial to run your application are used during development time only. Once you're ready to deploy your app, you'll want to take a look at the different deployment strategies for .NET Core apps and the dotnet publish command."